### DISCUSSION TITLE
[Proposal]: Doesn't use any dataset or model that does not compatible with Open Source

### DISCUSSION THREAD

**Discussion by wannaphong** [PyThaiNLP/pythainlp#725] [General]
I think we should doesn't use any dataset or model that does not compatible with Open Source in our project (PyThaiNLP). If the license that doesn't compatible with open source, we should ignore theirs datasets/models.

**Comment by bact** [comment]
+1

For data,
see some licenses that compatible with OKFN [Open Definition](https://opendefinition.org/) here https://resources.data.gov/open-licenses/


For code,
see all OSI-approved licenses here https://opensource.org/licenses

**Comment by wannaphong** [comment]
> +1
> 
> For data, see some licenses that compatible with OKFN [Open Definition](https://opendefinition.org/) here https://resources.data.gov/open-licenses/
> 
> For code, see all OSI-approved licenses here https://opensource.org/licenses

@bact  Do you think about LST20 Corpus? It is free for open source project only. I thinking remove all LST20 model and replace with Blackboard Treebank. https://github.com/PyThaiNLP/pythainlp/commit/18c8c509d3b785d197607d9e81fa77e35c2d5f53

**Comment by wannaphong** [comment]
Blackboard Treebank license is CC-BY. https://www.facebook.com/dancearmy/posts/10158423653343284?_rdc=1&_rdr

LST20 Corpus (free for non-commercial and open source only). https://www.facebook.com/dancearmy/posts/10157641945708284

**Comment by bact** [comment]
NC (non-commercial) clause makes things not fully open. OKFN Open Definition states that to be open it must also be open "for any purpose".

**Comment by wannaphong** [comment]
I think I will add the source that can’t use in the open source and ours project.

- ___- Website has legal requirements  (Do not forward any information from the web without allowed) and many dataset are CC-BY-SA-NC.
